app/femr/ui/views/partials/medical/dynamicField.scala.html

```diff
@@ -1,10 +1,60 @@
 @(value: java.lang.String, name: java.lang.String, size: java.lang.String, order: java.lang.Integer, tipe: java.lang.String, placeholder: java.lang.String, isReadOnly: Boolean, index: Integer)
 
-
 @if(size.toLowerCase == "medium") {
     <label for="@name">@name</label>
     <input name="tabFieldItems[@index].name" class="hidden" type="text" value="@name"/>
-    <input name="tabFieldItems[@index].value" type="text" placeholder="@placeholder" class="fInput customField" value="@value"@if(isReadOnly){readonly}/>
+    @if(tipe.toLowerCase == "yes/no") {
+        @if(name == "If no vaccinations, why not?"){
+            <div class="radio-responses">
+                <label class="radio-wrap">
+                    <input name="tabFieldItems[@index].value" type="radio" value="Cost" class="rInput" @if(isReadOnly) {readonly} @if(value == "cost") {checked}/>
+                    Cost
+                </label>
+                <br />
+                <label class="radio-wrap">
+                    <input name="tabFieldItems[@index].value" type="radio" value="No transport" class="rInput" @if(isReadOnly) {readonly} @if(value == "No transport") {checked}/>
+                    No transport
+                </label>
+                <br />
+                <label class="radio-wrap">
+                    <input name="tabFieldItems[@index].value" type="radio" value="Distrust health care providers" class="rInput" @if(isReadOnly) {readonly} @if(value == "Distrust health care providers") {checked}/>
+                    Distrust health care providers
+                </label>
+                <br />
+                <label class="radio-wrap">
+                    <input name="tabFieldItems[@index].value" type="radio" value="Did not know how" class="rInput" @if(isReadOnly) {readonly} @if(value == "Did not know how") {checked}/>
+                    Did not know how
+                </label>
+                <br />
+                <label class="radio-wrap">
+                    <input name="tabFieldItems[@index].value" type="radio" value="Did not need it" class="rInput" @if(isReadOnly) {readonly} @if(value == "Did not need it") {checked}/>
+                    Did not need it
+                </label>
+                <br />
+                <label class="radio-wrap">
+                    <input name="tabFieldItems[@index].value" type="radio" value="Other" class="rInput" @if(isReadOnly) {readonly} @if(value == "Other") {checked}/>
+                    Other
+                </label>
```

Comment:
This code is specific to the LSU team's request for options. It was easier/less risky to do custom view code versus adding support for editable radio options to custom tab fields